migrating_from_ocp_3_to_4/index.adoc

```diff
@@ -0,0 +1,86 @@
+[id="migration-from-version-3-to-4-overview"]
+= Migration from version 3 to 4 overview
+include::modules/common-attributes.adoc[]
+:context: migration-from-version-3-to-4-overview
+
+toc::[]
+
+{product-title} 4 clusters are different from {product-title} 3 clusters. {product-title} 4 clusters contain new technologies and functionality that result in a cluster that is self-managing, flexible, and automated. To learn more about migrating from {product-title} 3 to 4 see xref:../migrating_from_ocp_3_to_4/about-migrating-from-3-to-4.adoc#about-migrating-from-3-to-4[about migrating from OpenShift Container Platform 3 to 4].
+
+[id="mtc-3-to-4-overview-differences-mtc"]
+=== Differences between {product-title} 3 and 4
+Before migrating from {product-title} 3 to 4, you can check xref:../migrating_from_ocp_3_to_4/planning-migration-3-4.adoc#planning-migration-3-4[differences between {product-title} 3 and 4]. Review the following information:
+
+** Architecture
+** Installation and upgrade
+** Storage, network, logging, security, and monitoring considerations
+
+[id="mtc-3-to-4-overview-planning-network-considerations-mtc"]
+=== Planning network considerations
+Before migrating from {product-title} 3 to 4, review the following information to xref:../migrating_from_ocp_3_to_4/planning-migration-3-4.adoc#migration-preparing-networking[networking considerations]:
```

Comment:
I think you're either missing a word here or need to change a word (and I think this was my fault in my suggestion).

Right now it says "review the following information to networking considerations". Maybe it could be "review the following information on networking considerations"

Reply:
I would add that this lead-in sentence looks strange because the link in the sentence has nothing to do with DNS considerations. The link points to considerations, while the "DNS considerations" are [here](https://deploy-preview-40171--osdocs.netlify.app/openshift-enterprise/latest/migrating_from_ocp_3_to_4/planning-considerations-3-4.html). 

It is confusing because there are two different sections called networking or network considerations. That should be changed but not in this PR.

I think you should remove the 'networking considerations' link because it points to a subsection of "Differences between 3 and 4", which you already refer to earlier. Then link the DNS considerations to their locations on the "Network considerations" page.

Reply:
acked